en/docs/graphql-api.md

```diff
@@ -0,0 +1,90 @@
+         ![Enable preview features](../assets/img/feature-preview/enable-preview-feature.png){.cInlineImage-half}</br></br>
+
+## What is a GraphQL API?
+
+[GraphQL](https://graphql.org) is a query language for APIs to fetch and persist data. A GraphQL API is a category of APIs that supports this query language.
+
+A GraphQL API is an alternative for a REST API to address the following limitations experienced by REST API consumers:
+
+- A REST API may need to send multiple requests to several endpoints to retrieve all the required data. A GraphQL API can achieve this via a single query.
+
+    For example, a banking application has to fetch the phone number and the current account balance of an account holder. To achieve this, a REST API may need to send three requests to call three endpoints (e.g., `/customers/<id>` to fetch the initial data, `/customers/<id>/contactInfo` to fetch the phone number, and `/customers/<id>/balances` to fetch the current account balance).
+
+    A GraphQL API can retrieve all these details via one query similar to the sample given below:
+
+    ```
+    query {
+        customer (id: "SIL182882762") {
+            contactInfo {
+                phoneNo
+            }
+            balances(current)
+        }
+    }
+    ```
+    In this scenario, a single request by a REST API under-fetches data compared to a single request by a GraphQL API.
+
+- In some scenarios, a REST API can over-fetch data. In the same example given above, a request sent by REST API to the `/customers/<id>/contactInfo` may result in fetching all the contact information in the array such as the phone number, work phone number, address, email, etc., whereas the application only needs the phone number.
+
+    A GraphQL allows you to avoid this by writing queries with specific data requirements.
+
+By minimizing the number of requests it sends for a given purpose and avoiding over-fetching GraphQL API consumes a smaller amount of resources (such as the internet bandwidth, system memory, etc.). GraphQL APIs are therefore more economical. Therefore it is beneficial to consume GraphQL APIs for low-energy devices such as mobile devices compared to consuming REST APIs.
+
+
+## Develop
+
+To create a GraphQL you need to add a GraphQL component.
+
+![Add GraphQL component](assets/img/graphql/add-grapql-component..png){.cInlineImage-full}
+
+Choreo comes with the Web Editor which is an IDE designed based on Visual Studio Code for developers to implement API functionality. At present, developers can design GraphQL APIs in the Web Editor by writing code, and test the functionality via cURL commands before committing the design to a private/Choreo-managed GitHub repository. Once you commit the design, the GraphQL API is available in the Choreo Console where you can deploy, test, and observe it.
+
+![Implement graphql API](assets/img/graphql/implement-graphql-api.png){.cInlineImage-full}
```

Comment:
Here, let's only show the code view. At the moment, low code is not supported for GraphQL.

Example:
![image](https://user-images.githubusercontent.com/8376040/185735956-e51b0135-8638-45ad-864b-f406856f2fae.png)

Reply:
replaced the existing image with this one.